demo/client.cc

```diff
@@ -112,72 +103,117 @@ int main(int argc, char* argv[]) {
     request_data->set_filename(filename);
   }
 
-  if (!text_content.empty()) {
-    request.set_text_content(text_content);
-  } else if (!file_path.empty()) {
-    request.set_file_path(file_path);
+  if (!data.empty()) {
+    request.set_text_content(data);
+  } else if (!filepath.empty()) {
+    request.set_file_path(filepath);
   } else {
     std::cout << "[Demo] Specify text content or a file path." << std::endl;
     PrintHelp();
-    return 1;
+    return ContentAnalysisRequest();
   }
 
+  return request;
+}
+
+void DumpResponse(int position, const ContentAnalysisResponse& response) {
+  for (auto result : response.results()) {
+    auto tag = result.has_tag() ? result.tag() : "<no-tag>";
+
+    auto status = result.has_status()
+      ? result.status()
+      : ContentAnalysisResponse::Result::STATUS_UNKNOWN;
+    std::string status_str;
+    switch (status) {
+    case ContentAnalysisResponse::Result::STATUS_UNKNOWN:
+      status_str = "Unknown";
+      break;
+    case ContentAnalysisResponse::Result::SUCCESS:
+      status_str = "Success";
+      break;
+    case ContentAnalysisResponse::Result::FAILURE:
+      status_str = "Failure";
+      break;
```

Comment:
Should we have a "default" clause here? Same at line 158.

Reply:
Done.  I didn't think this would be needed since `status` is an enum type and these are all the possible values.  However I just checked the proto generated header and the type is `enum ContentAnalysisResponse_Result_Status : int { ... }` so seems like a default here is a good idea.